src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int randomIndex = new Random().nextInt(Colors.values().length);
+        String randomColor;
+        randomColor = "" + Colors.values()[randomIndex];
+        return randomColor;
+    }
```

Comment:
```suggestion
public class ColorSupplier {
   private Random random = new Random();
   
    public String getRandomColor() {
        int randomIndex = random.nextInt(Colors.values().length);
        String randomColor = Colors.values()[randomIndex].name();
        return randomColor;
    }
```

давай вынесем переменную Random на уровень класса, что б не создавать каждый раз новый экземпляр, когда мы вызываем метод getRandomColor()